lib/internal/test_runner/utils.js

```diff
@@ -434,6 +430,46 @@ function getCoverageReport(pad, summary, symbol, color, table) {
     return result;
   }
 
+  function getfilePathMultiline(string, width, pad, coverage) {
+    if (!table) return string;
+
+    const lines = [];
+    let currentLine = '';
+
+    for (const word of StringPrototypeSplit(string, '\\')) {
+      if (currentLine.length + word.length + 1 <= width) {
+        // If adding the next word fits in the current line, append it
+        currentLine += (currentLine.length > 0 ? '\\' : '') + word;
+      } else {
+        // If adding the next word exceeds the width, start a new line
+        ArrayPrototypePush(lines, currentLine);
+        currentLine = word;
+      }
+    }
+
+    // Add the last line if any
+    if (currentLine.length > 0) {
+      ArrayPrototypePush(lines, currentLine);
+    }
+
+    const truncatedLines = ArrayPrototypeMap(lines, (line) => StringPrototypeSlice(line, 0, width));
+
+    // Delete empty lines if any
+    for (let i = 0; i < truncatedLines.length; ++i) {
+      if (truncatedLines[i].length === 0) {
+        truncatedLines.splice(i, 1);
```

Comment:
Does this need primordials?

Reply:
Hey @RedYetiDev, I moved this PR to Draft while waiting for #54494 to be merged as at the moment, we don't have a test that covers the actual behavior.

Also, the current implementation in this PR doesn't solve the issue, so I'm going to rework it 😄

Regarding the primordials, thanks for the review: I'll replace all non-primordials 🚀